### PR TITLE
let initial click propagate through before changing visibility

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

**Comment by jquense** [comment]
I think this makes more sense here then in the RCW itself

**Comment by jquense** [comment]
dang tests

**Comment by taion** [comment]
What's the specific issue here? Do the dropdowns not open any more, or do they not close any more?

**Comment by jquense** [comment]
they don't open, because the `open` is pushed to the Menu before the click event triggering it bubbles up to the document. So the RCW sees the click and toggles it to closed immediately

File: test/DropdownButtonSpec.js [hunk 1]
```diff
@@ -122,11 +129,13 @@ describe('<DropdownButton>', () => {
     const node = ReactDOM.findDOMNode(instance);
 
     const buttonNode = ReactTestUtils.findRenderedDOMComponentWithTag(instance, 'BUTTON');
-    ReactTestUtils.Simulate.click(buttonNode);
+    await simulateClick(buttonNode);
```
Comment (jquense) [inline test/DropdownButtonSpec.js]: this feels like the best example of async/await being awesome :P

**Comment by jquense** [comment]
does this seem reasonable? I want get a fixed release out before folks start complaining. If we need to bikeshed more I can just revert the other PR as well

**Comment by taion** [comment]
Give me a sec.

**Comment by jquense** [comment]
fixed lower down in react-overlays 👍

**Pull request closed without merging**